content/en/blog/2023-03-23-rustls-new-features.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Josh Aas
+date: 2023-03-20T00:00:00Z
+slug: rustls-new-features
+title: "Rustls 0.21.0 Released With Exciting New Features"
+excerpt: "We’re incredibly excited about the latest release of Rustls, a memory safe TLS implementation."
+---
+
+We're incredibly excited about the latest release of [Rustls](https://github.com/rustls/rustls), a memory safe TLS implementation. This release has two major new features and a number of other improvements.
+
+The first big feature is support for TLS certificates containing IP addresses. Rustls can now be used to set up TLS connections addressed by IP rather than a domain name. This has been the most heavily requested feature for quite a while now and it's great to have it completed.
+
+The second big feature is support for [RFC8446 C.4 client tracking prevention](https://www.rfc-editor.org/rfc/rfc8446#appendix-C.4). This means that passive network observers will no longer be able to correlate connections.
```

Comment:
Maybe clearer if the connection between the RFC and the privacy implication is made more explicit by a mention of ticket reuse?

```suggestion
The second big feature is support for [RFC8446 C.4 client tracking prevention](https://www.rfc-editor.org/rfc/rfc8446#appendix-C.4). This means that passive network observers will no longer be able to correlate connections from ticket reuse.
```